xml/Microsoft.Extensions.Hosting.Systemd/SystemdNotifier.xml

```diff
@@ -61,7 +61,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Returns whether systemd is configured to receive service notifications.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value><see langword="true" /> if systemd is configured to receive service notifications; <see langword="false" /> otherwise.</value>
```